sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.h

```diff
@@ -828,6 +828,24 @@ static inline void _evtimer_add(void *ctx, int16_t type,
     evtimer_add_msg(&_nib_evtimer, event, target_pid);
 }
 
+/**
+ * @brief   Posts a message to the NIB bus of the interface
+ *
+ * @param[in] netif     Pointer to the interface
+ * @param[in] type      [Type of the event](@ref gnrc_ipv6_nib_event_t)
+ * @param[in] ctx       The context of the event
+ */
+static inline void _nib_bus_post(gnrc_netif_t *netif, int type, const void *ctx)
+{
+#ifdef MODULE_GNRC_NETIF_BUS
+    msg_bus_post(&netif->bus[GNRC_NETIF_BUS_NIB], type, ctx);
```

Comment:
Why is this a NIB bus btw? Last time I looked, the addresses are assigned to the `netif`, and except for the address validation being implemented in the NIB, there is no connection to the NIB now.

Reply:
Yea I think it's an IPV6_EVENT_BUS now.